documentation/guide-for-adding-banner-ads.md

```diff
@@ -0,0 +1,80 @@
+# Banner Ads Implementation Guide for all joomla.org sites
+This file describes how Joomla.org web masters should implement the ad serving code on our web properties.
+
+## ads.joomla.org
+We are running our own internal ([Revive](https://www.revive-adserver.com/)) ad server that should be serving all our ads. *We should not be serving ads directly from AdSense.*
+
+## Ad code
+The code to show ads is simple and in two parts.
+
+### General script
+```html
+<script async src="//ads.joomla.org/www/delivery/asyncjs.php"></script>
+```
+This code should be included **once** per page. The best way to include this code is most likely by pasting it into the template of the site. The code should be inside the `<body>` of the page near the `</body>` closing tag.
```

Comment:
Well even with the template switch you'd still need to get the ad specific snippet inserted somehow.  The module can load both the base `<script>` element and that snippet.  Without that snippet, loading the script from the ad server seems kinda pointless.

And honestly, we're talking about a module where the XML manifest is going to be longer than the PHP code 😆